app/controllers/katello/api/v2/host_subscriptions_controller.rb

```diff
@@ -35,6 +37,42 @@ def events
       respond_for_index :collection => collection
     end
 
+    api :POST, "/hosts/subscriptions/register", N_("Simulate a host subscription-manager registration")
```

Comment:
First, the rhsm controller is not a katello API, it is a candlepin API.

Second, there are a great number of existing (and future) use cases that rely on having an API method for creating and updating content hosts (or their equivaleng facets). Specifically, 'hammer csv' relies on it to populate "fake" systems as well as to import "real" Sat-5 systems via splice. There is also the case of registering against a prebuilt host model that is fully subscribed, etc.

Create and update of content hosts is very much needed.